### ISSUE TITLE
question on selective loss

### ISSUE THREAD

**Issue by lee-man** [geifmany/selectivenet#4]
`def selective_loss(y_true, y_pred):
            loss = K.categorical_crossentropy(
                K.repeat_elements(y_pred[:, -1:], self.num_classes, axis=1) * y_true[:, :-1],
                y_pred[:, :-1]) + lamda * K.maximum(-K.mean(y_pred[:, -1]) + c, 0) ** 2
            return loss`

Hi, just have a question on the selective loss.
Is output from selection head *y_pred[:, -1:]* a continous value after sigmoid?
After reading your paper, I think it should be the binary value after thesholding. Looking forward to your reply.

**Comment by sandruskyi** [comment]
Hi, did you resolve that? I have the same question @lee-man @geifmany
